<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupManagerForm</class>
 <widget class="QWidget" name="BackupManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backup Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Online ID / Username</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="accountBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Backup Type</string>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="backupComboBox">
         <item>
          <property name="text">
           <string>PS Vita Games</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PSP Games</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PSM Games</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PSOne Games</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PSP Savedatas</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Backups</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="columnCount">
      <number>1</number>
     </property>
     <column/>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="usageLabel">
       <property name="text">
        <string>Backup disk usage</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="FilterLineEdit" name="filterLineEdit">
       <property name="styleSheet">
        <string notr="true">FilterLineEdit { color:gray; font-style:italic; }</string>
       </property>
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save listing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
